### PR TITLE
Styling - Milestone 3

### PR DESCRIPTION
**Pull request by samgaco**
I have applied all these requirements:

* Added required styling by Microverse on the components, worked on the colors, display, widths, and heights of the different components.

### CODE REVIEW

File: src/components/ButtonPanel.js [hunk 1]
```diff
@@ -6,33 +6,32 @@ class ButtonPanel extends React.Component {
...
           <Button buttonName="1" />
           <Button buttonName="2" />
           <Button buttonName="3" />
           <Button buttonName="+" />
         </div>
-        <div className="group-5">
-          <Button buttonName="0" />
+        <div className="gbutton group-5">
+          <Button buttonName="0" color="grey" wide />
           <Button buttonName="." />
```
Comment (SunnyAsar) [inline src/components/ButtonPanel.js]: you missed `=` button

File: src/assets/style.css [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+  flex-direction: column;
+  width: 700px;
+}
+
+.button {
+  display: flex;
+}
+
+button {
+  width: 25%;
```
Comment (SunnyAsar) [inline src/assets/style.css]: the button heigh should fill its container.

File: src/assets/style.css [hunk 3]
```diff
@@ -0,0 +1,35 @@
+.bodyApp {
```
Comment (SunnyAsar) [inline src/assets/style.css]: Hi we have this now:

Screenshot 2019-12-03 at 8 16 32 AM


This is what our final design should look like:

Screenshot 2019-12-03 at 8 14 03 AM

without the missing button names of course.

Comment (SunnyAsar) [reply src/assets/style.css]: Let's send the color props of grey for all the button except the last in the group and the default props of orange can be used for the rest.

File: src/assets/style.css [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+  display: flex;
+  flex-wrap: wrap;
+  flex-direction: column;
+  width: 700px;
+}
```
Comment (shloch) [inline src/assets/style.css]: `margin: auto`

File: src/assets/style.css [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+
+.button {
+  display: flex;
+}
+
+button {
+  width: 25%;
```
Comment (shloch) [inline src/assets/style.css]: `font-size: {some higher value}`

**Comment by samgaco** [comment]
Thanks for the review, I am implementing those in the next branch I am working on for the milestone 4. I am merging this branch to development now :+1:

**Pull request merged**